recipes/pandas-alive/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pandas-alive" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pandas_alive/pandas_alive-{{ version }}.tar.gz
+  sha256: d88a8d81baafda50b77dfdc5bdf62b9c9d2f2d4ae53a031b168c244d7d80907b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - poetry >=0.12
+  host:
+    - poetry >=0.12
+    - python 3.*
+    - pip
+  run:
+    - python 3.*
```

Comment:
I don't think the `*` notation translates exactly to what is in the `pyproject.toml`. It would be:

```
requirements:
   host:
      - poetry >=0.12
      - pip
      - python >=3.6.1,<4.0.0
   run: 
      - python >=3.6.1,<4.0.0
      - pandas >=1.0.3,<2.0.0
      - matplotlib-base >=3.2.1,<4.0.0
      - attrs  >=19.3.0,<20.0.0
      - pillow >=7.1.2,<8.0.0
```

Also note using `matplotlib-base` vs `matplotlib`:
https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

Reply:
Thank you so much, this has been updated now. 😄